torch/autograd/profiler.py

```diff
@@ -712,6 +727,7 @@ def __repr__(self):
                 self.cpu_memory_usage,
                 self.cuda_memory_usage,
                 self.is_async,
+                self.is_remote,
```

Comment:
I don't have ctx here, sorry if this is a dump question. Can `is_async` and `is_remote` both be true? If they are mutually exclusive, should we make this field an enum instead of boolean? (Can come in followup PR)

Reply:
Technically remote events could be async as well, the definition of is_async is:
`is_async = start.thread_id() != record.thread_id()`